x/subspaces/keeper/feegrant.go

```diff
@@ -0,0 +1,166 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/desmos-labs/desmos/v4/x/subspaces/types"
+)
+
+// SaveUserGrant saves the given user grant
+func (k Keeper) SaveUserGrant(ctx sdk.Context, grant types.UserGrant) {
+	granteeAddr, err := sdk.AccAddressFromBech32(grant.Grantee)
+	if err != nil {
+		panic(err)
+	}
+	if !k.ak.HasAccount(ctx, granteeAddr) {
+		k.ak.SetAccount(ctx, authtypes.NewBaseAccountWithAddress(granteeAddr))
```

Comment:
Wer should add the following line to properly keep track of the account numbers: 
```
defer telemetry.IncrCounter(1, "new", "account")
```

We could even move these two lines in a new function named `createAccount` so that it's easier to manage. We would need this when adding a user to a group as well